products/cloudbuild/api.yaml

```diff
@@ -50,6 +50,10 @@ objects:
         description: |
           The unique identifier for the trigger.
         output: true
+      - !ruby/object:Api::Type::String
+        name: 'name'
```

Comment:
FYI: Reviewing in Terraform context
Adding a property to api.yaml will default to making it optional. However since this property is created by the API if it's not specified you will need to also have it marked as `Computed` in Terraform. If it's not Computed and a user upgrades to this version, subsequent `terraform apply` runs will attempt to override the `name` with an empty string as it will detect a difference between what had been specified in the config and what was set in the state from the API.

To do this you can set the property to `default_from_api: true` in the `terraform.yaml`. See `triggerTemplate.projectId` as an example.

Reply:
Thanks Chris! This makes a ton of sense. I'm adding this because creating multiple triggers at once sometimes results in them getting the same computed name. Thanks for reviewing and explaining how to add this default.

I've made the change (I think in the right way). 👍

Reply:
Great thanks, a much needed addition.